Feat/marketing site updates

paulatulis opened the pull request.

lottamus commented:
Deploy preview for *stoplightio* ready!

Built with commit ee8c1c199ec5c850e388d717a46cacd006bd7927

https://deploy-preview-1436--stoplightio.netlify.com

ksatx commented:
@paulatulis do you need review on #1422 #1424 #1427 #1431 still? Or are you deleting those branches and we'll work from here?

paulatulis commented:
> @paulatulis do you need review on #1422 #1424 #1427 #1431 still? Or are you deleting those branches and we'll work from here?

We will work from here. I'll close the other PRs. 

I've merged all separate PRs together into this one, so please leave comments for everything in this PR. 

Home: 
- Changed layout to be centered with gif placeholder over Hero divider
- Made gold option for tag because there was too much purple/blue. I can make background even lighter if it looks too flashy/is hard to read
- Need section header for SEO section at bottom

![Screen Shot 2020-03-18 at 4 46 39 PM](https://user-images.githubusercontent.com/47359669/77011029-f5bb4f80-6938-11ea-8134-a881047b95cb.png)


Enterprise:
- Placeholder hero image in place. Need stoplight admin to log in so that screenshot doesn't have my personal user info included. Would love Marc/Chris/Someone who has a good idea of the different graphs to decide on the best looking dependency graph. 
- Removed form and replaced with buttons to demo and workspaces 

<img width="1232" alt="Screen Shot 2020-03-18 at 4 46 16 PM" src="https://user-images.githubusercontent.com/47359669/77011037-fa800380-6938-11ea-9875-bfcd72888ec4.png">


To Do: 
- Collect all screenshots/gifs for landing pages
- Fix awkward images on About page/any other page that looked better with rounded feature images
- Collect/finalize all copy
- Pricing tooltips

Apologies if there are any other notes or action items I didn't address. Please comment or link to comment in this PR if you don't mind so I can keep everything in one place. Appreciate it!

paulatulis commented:
Home Page (from top, moving down):
- Removed 'log in' from banner
- Get Started goes to workspaces sign in flow
- Made gif video placeholder larger, but it looks awful. Need to put time into seeing how to make this look clear
- Reduced 6 cards to 4
- Still working on Solutions cards, so don't judge yet : P - unless we don't want them to look different from product cards...
- Removed SEO section at bottom, but need to probably put something else after case studies so it doesn't look weird
- Cannot remember if we decided to do away with case studies for now, but if anyone remembers, please comment and I'll remove them

paulatulis commented:
Pricing Page: 
- Evened out spacing above and below tier name and price on the top of each pricing card
- Section titles in feature grid now have 'API' appended to front
- Evened out top and bottom padding on sticky bar 
- Tooltip auto spacing works 

<img width="454" alt="Screen Shot 2020-03-23 at 9 55 18 AM" src="https://user-images.githubusercontent.com/47359669/77333708-d4b87d00-6cf1-11ea-85f5-68af4230c5d1.png">

ksatx commented:
First of all - those are some nice lookin tooltips, good job Paula!

paulatulis commented:
I made some additional updates to the cards on the home page. I think we should have an easily noticeable difference between product cards and painpoints/solutions, but the final design will depend on the copy (how long the descriptions are, if it makes sense to include icons, etc.)

@falsaffa when the content solidifies more, I can make additional adjustments, but would love your input in the meantime:


<img width="1017" alt="Screen Shot 2020-03-24 at 2 31 29 PM" src="https://user-images.githubusercontent.com/47359669/77469259-cdba6900-6ddc-11ea-811e-7c12c49a2a1b.png">

paulatulis commented:
Updates as of March 26 EOD: 

Home Page: 
- Larger placeholder gif that's the size of hero background image. Obviously this will change
- Separated product cards and pain point cards into two sections and updated copy (@falsaffa )

![Screen Shot 2020-03-26 at 4 56 05 PM](https://user-images.githubusercontent.com/47359669/77701048-6854ac80-6f83-11ea-9157-9522cad89d00.png)



Product & Solutions Landing Pages:
- Updated each feature section header copy (@falsaffa )

Pricing:
- Evened out spacing in top of pricing plan cards
- Evened out spacing in grid between section titles and features
- Updated FAQ layout 

![Screen Shot 2020-03-26 at 4 56 38 PM](https://user-images.githubusercontent.com/47359669/77701061-6e4a8d80-6f83-11ea-96a6-d79f452bf526.png)

paulatulis commented:
Changes since Friday: 

Homepage Hero
- Rounded edges on bottom 
- Stronger shadow on gif to raise off page 
- Rounded border of Hero

<img width="971" alt="Screen Shot 2020-03-30 at 2 00 49 PM" src="https://user-images.githubusercontent.com/47359669/77950918-e0c2b280-728e-11ea-8dcd-5790f1caaf60.png">


Solution Cards
- Learn more is same size as body text size 
- Should we add tag to card?
- Made CTA ‘read more about API design’
- Changed 'maintain' to 'increase'

<img width="886" alt="Screen Shot 2020-03-30 at 2 01 09 PM" src="https://user-images.githubusercontent.com/47359669/77950937-eae4b100-728e-11ea-8ed2-c52cdd31a5e3.png">


Images:
- Added a few new svgs for company logos but they are still a bit blurry. Still WIP.

<img width="790" alt="Screen Shot 2020-03-30 at 2 01 31 PM" src="https://user-images.githubusercontent.com/47359669/77950990-f89a3680-728e-11ea-8385-892abd1a4cdd.png">



Pricing FAQ:
- Moved FAQ carat to the left of the question
- Added divider above FAQ section 
- Made FAQ question font smaller
- Removed ‘don’t see your question…’
<img width="826" alt="Screen Shot 2020-03-30 at 2 02 06 PM" src="https://user-images.githubusercontent.com/47359669/77951077-1798c880-728f-11ea-8170-6c3c24546c44.png">

Pricing Grid: 
- Tried fitting grid to container, but it looks too spread out
- Inserted padding above 'API Design' section title
(ignore placement of top bar titles - they aren't aligned, this is just what the grid looks like when it takes up the full container)
![Screen Shot 2020-03-30 at 11 40 46 AM](https://user-images.githubusercontent.com/47359669/77951308-75c5ab80-728f-11ea-8023-6e7301132d56.png)

vs. what it currently looks like:

<img width="956" alt="Screen Shot 2020-03-30 at 1 59 26 PM" src="https://user-images.githubusercontent.com/47359669/77951323-79f1c900-728f-11ea-8f22-74edcd3c8cc5.png">

ksatx commented:
> Homepage Hero
> - Rounded edges on bottom
> - Stronger shadow on gif to raise off page
> - Rounded border of Hero

Looks great altogether, I especially like the return of the rounded border

> - Should we add tag to card?

I mocked up the tag - it's all going to the same link, so potentially another way to catch the eye? Personal opinion is it's a lot, and that the more important parts are a clear H2 that eliminates the need for a tag and addressing "Want to know how I can use Stoplight to promote team alignment - speak to the same thing on solution page". If/when we use the alternative card style, tags start to make more sense than icons. 

![image](https://user-images.githubusercontent.com/51092436/77960980-c2fd4980-729e-11ea-8ccf-f9e5f69a8cbb.png)

![image](https://user-images.githubusercontent.com/51092436/77961613-b9281600-729f-11ea-817c-f6121f45043a.png)


On the pricing FAQs, can you split them into two columns? Now that the right content is in there, and the carats are moved, I think it will look more balanced with three and three

ksatx commented:
Roughly thrown together, but you get the idea

![image](https://user-images.githubusercontent.com/51092436/77962628-8f6fee80-72a1-11ea-9a90-29dc3d2bc922.png)
![image](https://user-images.githubusercontent.com/51092436/77962642-972f9300-72a1-11ea-89d2-5dc6a4ecf5d2.png)

paulatulis commented:
Also add back testimonials with quote from Nicolas at Arkéa

paulatulis commented:
Added testimonials back with Nicolas from Arkea 

<img width="1243" alt="Screen Shot 2020-04-01 at 4 25 50 PM" src="https://user-images.githubusercontent.com/47359669/78188214-76984200-7435-11ea-856a-1619f275dec9.png">

Updated FAQ layout to 2 columns
<img width="1200" alt="Screen Shot 2020-04-01 at 4 25 38 PM" src="https://user-images.githubusercontent.com/47359669/78188225-7bf58c80-7435-11ea-9ee2-b2fb83990543.png">

paulatulis commented:
TO DO: 
remove careers from nav bar

paulatulis commented:
Pricing page accounts for four plans with updated feature grid and plan cards. 

Still to do: 
- finalize the bullets for each plan card at the top
- tool tips

<img width="1269" alt="Screen Shot 2020-04-30 at 1 34 24 PM" src="https://user-images.githubusercontent.com/47359669/80746447-8617b300-8ae7-11ea-8ac1-babbefc41ff0.png">
<img width="1314" alt="Screen Shot 2020-04-30 at 1 34 30 PM" src="https://user-images.githubusercontent.com/47359669/80746448-86b04980-8ae7-11ea-92a0-1653ec788bc9.png">

paulatulis commented:
@ksatx image section added https://deploy-preview-1436--stoplightio.netlify.app/enterprise/
related to [1451](https://github.com/stoplightio/stoplight.io/issues/1451)

philsturgeon commented on netlify/landings/docs.yaml:
```diff
@@ -1,75 +1,63 @@
+      title: Avoid Integration Hand Holding
     - description: >-
-        Stoplight Docs uses a powerful Markdown engine built on top of the CommonMark spec. Enhance your documentation with embedded models, code snippets, call outs and more to give your users the best experience.
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```
Lorem ipsum!

ksatx replied:
Ah, yeah, so our main dev branch for these updates is now #1467 - the PR's are a bit out of hand at the moment
